src/bundle/Resources/public/scss/_bulk-operation-failed-modal.scss

```diff
@@ -0,0 +1,25 @@
+.ez-bulk-operation-failed-modal {
+    .modal-dialog {
+        max-width: 60vw;
+    }
+
+    .modal-header {
+        background-color: $ez-color-base-pale;
+
+        .modal-title {
```

Comment:
Do you need to nest this selector and the one below?

Reply:
Hmm, we nest this selector in other places, but I checked it and we don't have to do this. 
Also, I modeled this on notifications modal, which has separate file with CSSes styling modal, but now I realised that we are storing modals CSSes in: https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/public/scss/_modals.scss
so these styles should probably be moved to this file?